notebooks/data-sources/TestGrid/metrics/pct_fixed_each_ts.ipynb

```diff
@@ -2,20 +2,24 @@
  "cells": [
```

Comment:
<p>See the import cell example from <code>correlated_failures.ipynb</code>  below.   If we import from the metrics template we don't need to include the sys.path  stuff and the "#noqa" flags can be removed.  Also, you need t add the <code>load_dotenv(find_dotenv())</code>  too.  </p><pre class="ql-syntax" spellcheck="false">
# Import libraries
import gzip
import json
import os
import numpy as np
import pandas as pd
import datetime

from ipynb.fs.defs.metric_template import decode_run_length
from ipynb.fs.defs.metric_template import CephCommunication
from ipynb.fs.defs.metric_template import save_to_disk, read_from_disk
from dotenv import load_dotenv, find_dotenv

load_dotenv(find_dotenv())
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>